### PR TITLE
JBPM-5467: Stunner - UXD palette integration

### PR DESCRIPTION
**Pull request by pefernan**
This PR adds support to i8n for the stunner models literals (DefinitionSet (Description literal), Definition (Title, Description & Category literals), PropertySet (Name literal) & Property (Caption & Description literal).

The same literals defined on the bundles will be used to generate the Properties forms.

It includes some code required for https://issues.jboss.org/browse/JBPM-5622

@romartin would you like to take a look?

### CODE REVIEW

**Comment by pefernan** [comment]
@tsurdilo have this PR in mind if you're creating new nodes / properties:
- Notice how the labels are registered on the properties files (https://github.com/pefernan/kie-wb-common/blob/dc6c395f92d7b43ede24f3d54abc78ea2bf1ecbb/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/resources/org/kie/workbench/common/stunner/bpmn/resources/i18n/StunnerBPMNConstants.properties) using the FQN of the model and the corresponding suffix (title, name, description, category...) depending on the model type.
- Notice that there is no more i18n config on @FormDefinitions (only on stunner forms) and also there are no more labelKeys on the @FormFields (https://github.com/pefernan/kie-wb-common/blob/f5eb9ad4327e6663aea07c1f0a504b5b2f612604/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/java/org/kie/workbench/common/stunner/bpmn/definition/UserTask.java)
- Notice that each class annotated with @FieldDefinition should use the LabelMode.OVERRIDE_I18N_KEY and mark the label property as @FieldLabel (https://github.com/pefernan/kie-wb-common/blob/dc6c395f92d7b43ede24f3d54abc78ea2bf1ecbb/kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-api/src/main/java/org/kie/workbench/common/stunner/bpmn/definition/property/assignee/Actors.java)
- Notice that on PropertySets the propoertySetName is marked with the @FieldLabel annotation

**Comment by pefernan** [comment]
jenkins retest this.

**Comment by pefernan** [comment]
jenkins retest this.

**Comment by pefernan** [comment]
jenkins retest this.

**Comment by pefernan** [comment]
jenkins retest this.

File: kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-client-common/src/main/java/org/kie/workbench/common/stunner/core/client/definition/adapter/binding/ClientBindablePropertyAdapter.java [hunk 1]
```diff
@@ -22,6 +22,7 @@
 import org.kie.workbench.common.stunner.core.definition.adapter.binding.BindableAdapterUtils;
 import org.kie.workbench.common.stunner.core.definition.adapter.binding.BindablePropertyAdapter;
 import org.kie.workbench.common.stunner.core.definition.property.PropertyType;
+import org.kie.workbench.common.stunner.core.i18n.StunnerTranslationService;
 
 class ClientBindablePropertyAdapter extends AbstractClientBindableAdapter<Object> implements BindablePropertyAdapter<Object, Object> {
```
Comment (hasys) [inline kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-client-common/src/main/java/org/kie/workbench/common/stunner/core/client/definition/adapter/binding/ClientBindablePropertyAdapter.java]: Hi @pefernan,

great job! Just one question: what do you think. This class and all classes from this package have no tests. Are they worth to test?

Comment (pefernan) [inline kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-client-common/src/main/java/org/kie/workbench/common/stunner/core/client/definition/adapter/binding/ClientBindablePropertyAdapter.java]: @hasys tests added.

@romartin also included the last CSS change to make avoid palette movements on mouse over.

Comment (romartin) [inline kie-wb-common-stunner/kie-wb-common-stunner-core/kie-wb-common-stunner-commons/kie-wb-common-stunner-client-common/src/main/java/org/kie/workbench/common/stunner/core/client/definition/adapter/binding/ClientBindablePropertyAdapter.java]: Hey @pefernan , thanks for the tests added! Verified new CSS, all fine :)

**Pull request merged**